subscription-engine/kafka-ws/test/unit/KafkaConsumers.test.js

```diff
@@ -48,14 +48,31 @@ describe('Testing KafkaWSConsumers - works fine', () => {
       unregisterCallback: jest.fn()
         .mockImplementationOnce(() => () => Promise.resolve()),
       getStatus: jest.fn()
-        .mockImplementationOnce(() => Promise.resolve({ connected: true })),
+        .mockImplementationOnce(() => Promise.resolve({ connected: true }))
+        .mockImplementationOnce(() => Promise.resolve({ connected: false }))
+        .mockImplementationOnce(() => Promise.reject()),
```

Comment:
Shouldn't it be made in the beginning of the test that will use these returns? If the tests are run out of order, this could cause non deterministic behavior, isn't it?